packages/ui/src/components/ColumnManager.tsx

```diff
@@ -152,9 +122,13 @@ export function ColumnManager<RowDataModel>({ table }: { table: Table<RowDataMod
         }
       }}
     >
-      <Dropdown.Toggle variant="outline-secondary" id="column-manager-button">
-        <i class="bi bi-view-list me-2" aria-hidden="true" />
-        View
+      <Dropdown.Toggle
+        variant="tanstack-table"
+        id={id}
+        // eslint-disable-next-line @eslint-react/no-forbidden-props
+        className="tanstack-table-focusable-shadow"
```

Comment:
This is interesting: `Dropdown.Toggle` internally merges a `className` prop with its own computed classname, and then spreads all other props into the component it rendered. This means that when I used a `class` prop here, it took precedence over the computed `className`, which left this without the `dropdown-toggle` prop. The fix of course is just to use `className` instead here.

I don't know if there's any bigger lesson to be learned here.